### DISCUSSION TITLE
Setting up a local environment

### DISCUSSION THREAD

**Discussion by retsofoster** [TEAMMATES/teammates#12089] [Help requests] [answered]
I posted in the wrong channel (sorry)... I am having trouble getting the environment to show up via http://localhost:4200/web/front/home the code is there if I inspect element but the screen is blank.

**Comment by zhaojj2209** [comment, marked as answer]
Likely an issue with your browser, this issue seems to be local to your machine and as such we are unable to help you troubleshoot.
